docs/examples/ex41.py

```diff
@@ -0,0 +1,47 @@
+"""Mixed meshes.
+
+There exists a preliminary support for mixed meshes, i.e. meshes combining
+multiple element types such as triangles and quadrilaterals.  This is mostly
+because the implementation of :func:`~skfem.assembly.asm` supports lists of
+:class:`~skfem.assembly.basis.AbstractBasis` objects and the resulting matrices
+of the multiple assembly operations is combined in the end.
+
+The working principle is that a list of meshes is created by the user, one for
+each element type.  Then the finite element basis is created separately for
+each mesh.  Finally, the list of basis objects is passed on to
+:func:`~skfem.assembly.asm`.
+
+This example demonstrates how a mesh generated by Gmsh, consisting of triangles
+and quadrilaterals, can be used to solve the Poisson problem with unit load.
+Currently the user must verify that the finite element bases are compatible
+with one another.  Moreover, the mixed meshes work properly only with finite
+elements that have nodal degrees-of-freedom.  Support for higher-order elements
+on mixed meshes is work-in-progress.
+
+"""
+from skfem import *
+from skfem.models import laplace, unit_load
+
+fname = 'docs/examples/meshes/mixedtriquad.msh'
+out = ['cell_sets_dict']  # read boundary nodes from meshio
+m = [
+    Mesh.load(fname, force_meshio_type='triangle'),
+    Mesh.load(fname, force_meshio_type='quad', out=out),
+]
+e = [ElementTriP1(), ElementQuad1()]
+basis = [Basis(*me) for me in zip(m, e)]
```

Comment:
This could be written slightly more concisely as
```python
basis = list(map(Basis, m, e))
```
but it hardly makes a difference.  Maybe yours is more pythonic anyway.

What is fantastic though is how concise clear and pythonic (and &lsquo;skfemic&rsquo;!) this whole example is!  A great advance on the long-standing #412.

Reply:
Decided to use this, I didn't know that `map` works like this for non-functions. Cool!